src/ontology/uberon-edit.obo

```diff
@@ -224531,6 +224531,148 @@ relationship: dc-contributor https://orcid.org/0000-0001-6677-8489 ! Aleix Puig-
 relationship: located_in UBERON:8600025 ! intervillous space
 property_value: dcterms-date "2024-04-15T10:05:35Z" xsd:dateTime
 
+[Term]
+id: UBERON:8600108
+name: proximal ureter
+def: "A segment of an ureter that extends from the renal pelvis to the common iliac artery bifurcation (to form the internal and external iliac arteries)." [PMID:30422575]
```

Comment:
A segment of **the** ureter instead of **an**

Reply:
It's true I haven't been consistent, I will fix it, thank you!